app/bundles/CampaignBundle/Assets/css/campaign.css

```
@@ -107,6 +107,18 @@
     color: #47535f;
 }
 
+.jtk-label--success {
+    border-color: var(--support-success);
+    background-color: var(--support-success);
+    color: var(--text-inverse);
+}
+
+.jtk-label--error {
+    border-color: var(--support-error);
+    background-color: var(--support-error);
+    color: var(--text-inverse);
+}
+
```

```suggestion

.campaign-event-stat {
    position: absolute;
    bottom: 0;
    right: 0;
    padding: 4px 8px;
}
```

The CSS styling for my changes in the Twig template